server/app.js

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+
+const app = express();
+const port = process.env.PORT || 3000;
+
+app.get('/', (req, res) => {
+  res.send('Welcome to Politico!');
+});
```

Comment:
All responses from the API should be in JSON format

Reply:
Thanks... I corrected it in PR #6. Can I merge that one too then?